compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/codegen/FunctionCodegen.kt

```diff
@@ -170,7 +170,7 @@ class FunctionCodegen(
 
         return getVisibilityAccessFlag() or modalityFlag or functionDeprecationFlags or
                 (if (isStatic) Opcodes.ACC_STATIC else 0) or
-                (if (isVararg) Opcodes.ACC_VARARGS else 0) or
+                (if (isVararg && !isBridge) Opcodes.ACC_VARARGS else 0) or
```

Comment:
Another minor nitpick, but I'd prefer if `&& !isBridge` was a part of the `isVararg` value declaration, just to avoid misusing `isVararg` between its declaration and this usage

Reply:
Done, thanks!